modules/lambdas/main.tf

```diff
@@ -393,6 +393,30 @@ resource "aws_s3_bucket_notification" "post_copy_request_to_queue_trigger" {
 
 }
 
+# orca_catalog_reporting_dummy - Returns reconcilliation report sample data
+# ==============================================================================
+resource "aws_lambda_function" "orca_catalog_reporting_dummy" {
+  ## REQUIRED
+  function_name = "${var.prefix}_orca_catalog_reporting_dummy"
+  role          = module.restore_object_arn.restore_object_role_arn
+
+  ## OPTIONAL
+  description      = "Returns reconcilliation report sample data."
+  filename         = "${path.module}/../../tasks/orca_catalog_reporting_dummy/orca_catalog_reporting_dummy.zip"
+  handler          = "orca_catalog_reporting_dummy.handler"
+  memory_size      = var.orca_recovery_lambda_memory_size
+  runtime          = "python3.7"
+  source_code_hash = filebase64sha256("${path.module}/../../tasks/orca_catalog_reporting_dummy/orca_catalog_reporting_dummy.zip")
+  tags             = local.tags
+  timeout          = var.orca_recovery_lambda_timeout
```

Comment:
We may want to use `var.orca_ingest_lambda_timeout` here. Since reconciliation is more associated with ingest. Long term we may want some other variables but for this exercise either will work.

Reply:
Fixed